docs/contributors-guide.md

```diff
@@ -1,34 +1,30 @@
+
+## Advice for new contributors
+Contributing to an OSS project requires you to figure out things on your own when you can, and seek help from the right resources (Google, StackOverflow, [troubleshooting guide](troubleshooting-guide.md), [issue tracker](https://github.com/TEAMMATES/teammates/issues), etc.). To become a TEAMMATES contributor, you need to start honing those skills.
+
+To help you with that, we have created a series of easy tasks you can should start off with.
+
+**Table of Contents**
 
 * [Knowledge required](#knowledge-required)
 * [Getting started](#getting-started)
 * [Orientation task list](#orientation-task-list)
+* [Roles](#roles)
 * [Contacting us](#contacting-us)
 
-## Knowledge required
+## Learning the basics
 
-Although TEAMMATES uses many tools and technologies, you need not know all of them before you can contribute. The diagram below shows which technologies you need to learn to contribute in different roles.
+Although TEAMMATES uses many tools and technologies, you need not know all of them. The diagram below shows which technologies you need to learn to contribute in different [roles](#roles).
 
 ![RolesAndTechnologies.png](images/RolesAndTechnologies.png)
 
-As you can see, some roles do not involve learning any technologies at all.
-Note that these role are not fixed or formally assigned; it is simply for guidance only.
-
-Roles:
-
-* **Tester**: Tests the App manually and reports bugs or suggestions for enhancements in the issue tracker.
-* **Copy editor**: Helps in improving documentation.
-* **Web page developer**: Works on static web pages, such as those used in the TEAMMATES product web site.
-* **Test developer**: Works on automating system tests.
-* **Frontend developer**: Works on the frontend of the App that is generated dynamically from the server side.
-* **Backend developer**: Works on the backend logic of the App, including data storage.
+As you may notice, Git is essential to all roles. A good place to learn git is [here](https://try.github.io/levels/1/challenges/1).
 
 ## Getting started
 
-Contributing to an OSS project requires you to figure out things on your own when you can, and seek help from the right resource (Google, StackOverflow, troubleshooting guides, issue tracker, etc.) when you cannot. To become a TEAMMATES contributor, you need to start honing those skills.<br>
-To help you with that, we have created a sequence of tasks you can try to complete. Try to complete as many of them as you can, in the order they are listed.<br>
 Of course we are happy to guide you if you encounter any difficulties when doing these tasks; we have provided a [troubleshooting guide](troubleshooting-guide.md) for some commonly faced problems. Failing that, you may post a help request in our [issue tracker](https://github.com/TEAMMATES/teammates/issues).
```

Comment:
Seems a bit abrupt; the old version without the earlier lines removed might be better.